src/UI/HTMLTooltip.js

```diff
@@ -1,6 +1,6 @@
 import { safeExecute, addInlineStyles, whenIdle } from '../autofill-utils.js'
 import { getSubtypeFromType } from '../Form/matching.js'
-import { CSS_STYLES } from './styles/styles.js'
+import CSS_STYLES from './styles/autofill-tooltip-styles.css'
```

Comment:
esbuild supports VERY modern css stuff, so we can just import the files like that and configure it to be a string.